scripts/util/create-kubernetes-binaries-iso.sh

```diff
@@ -87,6 +91,14 @@ if [ $? -ne 0 ]; then
 fi
 mkdir -p "${working_dir}/docker"
 output=`${k8s_dir}/kubeadm config images list --kubernetes-version=${RELEASE}`
+
+# Don't forget about the yaml images !
+for i in ${network_conf_file} ${dashboard_conf_file}
+do
+  images=`grep "image:" $i | cut -d ':' -f2- | tr -d ' ' | tr -d "'"`
+  output=`printf "%s\n" ${output} ${images}`
```

Comment:
`printf` has only one specifier in the format and two arguments. What is intended here?

Reply:
It's a simpler way of doing `printf "%s\n%s\n" ${output} ${images}` :D

Reply:
ok, never encountered that before, tnx